AlembicImporter/Packages/com.unity.formats.alembic/Editor/Misc/BoolDrawer.cs

```diff
@@ -6,8 +6,15 @@ namespace UTJ.Alembic
     [CustomPropertyDrawer(typeof(Bool))]
     class BoolDrawer : PropertyDrawer
     {
+        private BoolDrawer() { }
```

Comment:
warning suggested to add private constructor as class is never instantiated. Will have to make sure this doesn't cause issues